addons/pos_self_order/controllers/manifest_controller.py

```diff
@@ -0,0 +1,75 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import json
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class PosManifest(http.Controller):
+
+    @http.route('/pos-self/manifest/<config_id>/<access_token>', type='http', auth='public', methods=['GET'])
+    def posmanifest(self, config_id, access_token):
+        """ Returns a posmanifest describing the metadata associated with a web application.
+        Using this metadata, user agents can provide developers with means to create user
+        experiences that are more comparable to that of a native application.
+        """
+        company_name = request.env.company.name
+        config = request.env['pos.config'].search([('id', '=', config_id)], limit=1)
```

Comment:
```suggestion
        config = request.env['pos.config'].search([('id', '=', config_id)], limit=1)
        company_name = config.company_id.name
```

derive the company name from the company of the config.